.circleci/config.yml

```diff
@@ -22,7 +24,9 @@ jobs:
   Rust and Foreign Language tests:
     docker:
       - image: rfkelly/uniffi-ci:latest
+    resource_class: small
     steps:
+      - run: cat ~/.profile >> $BASH_ENV
```

Comment:
I added a `.circleci/config.yaml` directly on master so that circle would start building things, but I'd welcome any feedback on it as part of this PR.